src/game/client/components/render_layer.cpp

```diff
@@ -1017,6 +1027,102 @@ void CRenderLayerQuads::Init()
 	RenderLoading();
 }
 
+void CRenderLayerQuads::CalculateClipping()
+{
+	// calculate clipping if not too expensive
+	if(!m_Grouped)
+		return;
+
+	// calculate envelope position offsets
+	ivec2 aEnvOffset[] = {{0, 0}, {0, 0}};
+	if(m_QuadRenderGroup.m_PosEnv != -1)
```

Comment:
```suggestion
	if(m_QuadRenderGroup.m_PosEnv >= 0)
```

Consistent with other checks.